src/utils/config.py

```diff
@@ -40,6 +40,14 @@
     "schema": os.getenv('SNOWFLAKE_DATA_RETENTION_SCHEMA')
 }
 
+MYSQL_PUBLISHER_CONNECTION_DICT = {
+    "host": os.getenv("POCKET_PUBLISHER_DATABASE_HOST"),
+    "user": os.getenv("POCKET_PUBLISHER_DATABASE_USER"),
+    "db_name": os.getenv("POCKET_PUBLISHER_DATABASE_DBNAME"),
+    "port": int(os.getenv("POCKET_PUBLISHER_DATABASE_PORT")),
+    "password": os.getenv('POCKET_PUBLISHER_DATABASE_PASSWORD'),
+}
```

Comment:
POCKET_PUBLISHER_DATABASE_PORT = os.getenv("POCKET_PUBLISHER_DATABASE_PORT")
```suggestion
MYSQL_PUBLISHER_CONNECTION_DICT = {
    "host": os.getenv("POCKET_PUBLISHER_DATABASE_HOST"),
    "user": os.getenv("POCKET_PUBLISHER_DATABASE_USER"),
    "db_name": os.getenv("POCKET_PUBLISHER_DATABASE_DBNAME"),
    "port": int(POCKET_PUBLISHER_DATABASE_PORT) if POCKET_PUBLISHER_DATABASE_PORT else None,
    "password": os.getenv('POCKET_PUBLISHER_DATABASE_PASSWORD'),
}
```